.gitignore

```diff
@@ -60,3 +60,5 @@ java/build/
 java/module_phase_1.xml
 java/testreport/
 testreport/
+.idea/
+java/Phase 1.iml
```

Comment:
Why? These two shouldn't need to be here. `Phase 1.iml` should be updated when we add components, etc.